px-alert-message.html

```diff
@@ -59,22 +59,22 @@
       </template>
       <div class="message-column flex flex--middle">
         <div class="message-container">
-          <div class$="{{type}} message collapsed-message" id="message">
+          <div class$="[[_getMessageClassNames(type, expanded)]]" id="message">
             <span class="title">{{ messageTitle }}</span>
             <span>{{ message }}</span>
           </div>
-          <span id="showMoreButton" class="actionable visuallyhidden show-more">[[localize(_showBtnText)]]</span>
+          <span id="showMoreButton" class="actionable show-more" hidden$="[[!overset]]">[[_showBtnText]]</span>
         </div>
       </div>
       <template is="dom-if" if="{{ action }}">
         <div class="action">
           <template is="dom-if" if="{{ _isTrue(action, 'dismiss') }}">
-            <button class="btn btn--bare dismiss" on-click="_dismiss">
+            <button id="dismissButton" class="btn btn--bare dismiss" on-click="_action">
               <px-icon icon="px-nav:close"></px-icon>
             </button>
           </template>
           <template is="dom-if" if="{{ !_isTrue(action, 'dismiss') }}">
-            <button class="btn btn--tertiary" on-click="_action">[[localize(_actionText)]]</button>
+            <button id="actionButton" class="btn btn--tertiary" on-click="_action">[[localize(_actionText)]]</button>
```

Comment:
See comment above. I moved this to `this.localize(TEXT)` in the `_computeActionText()` method to be consistent for now, LMK if I should change that.